### DISCUSSION TITLE
How to disable Intercepting Routes (Parallel Routes) on mobile

### DISCUSSION THREAD

**Discussion by kungfu321** [vercel/next.js#54671] [App Router]
Hello everyone!

I hope you're doing well. I am writing to discuss an issue I have encountered while using the new features `Intercepting Routes` and `Parallel Routes` in Next.js 13. I have thoroughly reviewed the documentation, tested the "Nextgram" project, and watched videos to understand this new feature.

I'm using `Intercepting Routes` to show a login modal it's working well. But I want to on mobile it will redirect to `/login` page not a modal. 
So can we disable `Parallel Routes` or how to config that.
Thank you.

**Comment by duoluodexiaoxiaoyuan** [comment]
I meet a problem. I use Intercepting Routes  don't find url. 
I  don't know how to Google
this is my repo: https://github.com/duoluodexiaoxiaoyuan/mynextproject

**Comment by boredjoker** [comment]
anything update here?
just same question, is there a way to disable Parallel Routes on mobile?
and I found Parallel Routes is not work with reactCompiler
